.github/workflows/ci.yaml

```diff
@@ -12,6 +12,7 @@ jobs:
         activerecord:
         - "52"
         - "60"
+        - "61"
```

Comment:
Do we want to add 7.0 as a another entry in the matrix, but allowed to fail?  If so, the way I did that elsewhere was like this:

```yaml
      matrix:
        ruby-version:
        - '2.7'
        activerecord:
        - "52"
        - "60"
        - "61"
        allow-failure: false
        include:
        - activerecord: "70"
          allow-failure: true
```

Then down below (in each of the test sections)

```yaml
      - name: Run SQLite tests
        continue-on-error: ${{ matrix.allow-failure }}
        env:
          DB: sqlite3
          CC_TEST_REPORTER_ID: ${{ secrets.CC_TEST_REPORTER_ID }}
```

Reply:
I was definitely leaning towards no here. We are quite far away from 7.0

Do you want me to put it in anyway?

Anything else you want before we push this out?

Reply:
I see no value in testing 7.0 if we can't even get useful failures or the error/failures are so numerous, it's too hard to read. I'll leave that up to you to determine how close we are to supporting 7.0

Reply:
not close enough.

Reply:
sounds good...no need for 70 then :)